### ISSUE TITLE
[Bug]: UI not showing escaped JSON correctly

### ISSUE THREAD

**Issue by vdurante** [jaegertracing/jaeger-ui#1570]
### What happened?

An event with escaped JSON gets unescaped when displaying in the UI.

I have a log value such as:

`
{
    "name" : "Marcus",
    "value": "{ \"height\" : 193 }"
}`

The value property above, as you can see, is a JSON that got serialized and escaped.

When the trace is built, it then serializes the value property again:

This becomes the following string on my trace:

`
"{\"name\":\"Marcus\",\"value\":\"{\\\"height\\\":139}\"}"
`

Then, when Jager UI displays this JSON, it deserializes it multiple times creating a non-valid JSON value on the ui

### Steps to reproduce

Use the following file on JSON import of Jaeger UI

[json test.txt](https://github.com/jaegertracing/jaeger-ui/files/12017156/json.test.txt)

Please rename it to .json

### Expected behavior

I expect the interface to properly deserialize the JSON and producing a valid output

### Screenshot

Looks correct when minimized:

<img width="472" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/5401232/75ecf496-6a9f-43c4-913c-094fda46cb92">

But when I expand:

<img width="472" alt="image" src="https://github.com/jaegertracing/jaeger-ui/assets/5401232/6f729c0f-bd75-4e65-9116-09163ae0c3f4">

### Jaeger backend version

jaegertracing/jaeger-collector:1.36

### Operating system

Mac OS

### Deployment model

Kybernetes / Terraform

**Comment by anj20** [comment]
I want to solve the bug @vdurante can u assign this to me?

**Comment by yurishkuro** [comment]
@anj20 see https://github.com/jaegertracing/jaeger/blob/main/CONTRIBUTING_GUIDELINES.md#assigning-issues

**Comment by yurishkuro** [comment]
@vdurante please post what you expect to see in the expanded view

**Comment by vdurante** [comment]
@yurishkuro a wellformed JSON

So, instead of 

```json
 { 
    "name" : "Marcus", 
    "value: "{ "height" : 193 }" 
}
```

Should be


```json
 { 
    "name" : "Marcus", 
    "value: "{ \"height\" : 193 }" 
}
```

**Comment by yurishkuro** [comment]
Fair enough. Need to be careful not to introduce XSS bugs that already was fixed once in the JSON rendering.

**Comment by yurishkuro** [comment]
I think the fix itself may be simple enough, so tagging good-first-issue

**Comment by anj20** [comment]
Can u walk me through the file structure?

**Comment by Xaid-vfx** [comment]
Hey @yurishkuro ,
I would like to resolve this issue. Could you assign me?

**Comment by yurishkuro** [comment]
should be fixed by #1724
